### ISSUE TITLE
windows_native_app/7.4.184 crashes after selecting time entry name from autocomplete dropdown

### ISSUE THREAD

**Issue by maritoggl** [toggl/toggldesktop#2508]
User reports that app seems to be crashing when selecting a time entry from autocomplete dropdown. Logs available in case/730385.

**Comment by NghiaTranUIT** [comment]
Related #2872

**Comment by IndrekV** [comment]
Description from user
> app fail silently closes after i type already created time entry name and click on one of the founded 

Error log
```
System.ArgumentOutOfRangeException: Index je mimo rozsah. Index musí být nezáporný a musí být menší než velikost kolekce.
Název parametru: index
   v System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   v TogglDesktop.AutoCompletion.AutoCompleteController.SelectNext()
   v TogglDesktop.AutoCompletionPopup.textboxOnPreviewKeyDown(Object sender, KeyEventArgs e)
   v System.Windows.Input.KeyEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   v System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   v System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   v System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   v System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   v System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   v System.Windows.UIElement.RaiseEvent(RoutedEventArgs args, Boolean trusted)
   v System.Windows.Input.InputManager.ProcessStagingArea()
   v System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   v System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   v System.Windows.Interop.HwndKeyboardInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawKeyboardActions actions, Int32 scanCode, Boolean isExtendedKey, Boolean isSystemKey, Int32 virtualKey)
   v System.Windows.Interop.HwndKeyboardInputProvider.ProcessKeyAction(MSG& msg, Boolean& handled)
   v System.Windows.Interop.HwndSource.CriticalTranslateAccelerator(MSG& msg, ModifierKeys modifiers)
   v System.Windows.Interop.HwndSource.OnPreprocessMessage(Object param)
   v System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   v System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   v System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   v System.Windows.Threading.Dispatcher.Invoke(DispatcherPriority priority, Delegate method, Object arg)
   v System.Windows.Interop.HwndSource.OnPreprocessMessageThunk(MSG& msg, Boolean& handled)
   v System.Windows.Interop.HwndSource.WeakEventPreprocessMessage.OnPreprocessMessage(MSG& msg, Boolean& handled)
   v System.Windows.Interop.ThreadMessageEventHandler.Invoke(MSG& msg, Boolean& handled)
   v System.Windows.Interop.ComponentDispatcherThread.RaiseThreadMessage(MSG& msg)
   v System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   v System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   v System.Windows.Application.RunDispatcher(Object ignore)
   v System.Windows.Application.RunInternal(Window window)
   v System.Windows.Application.Run(Window window)
   v TogglDesktop.Program.Main()
```

**Comment by IndrekV** [comment]
closed with #2887
